.github/workflows/verify.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+name: Verify
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x]
```

Comment:
Do we actually need to test on 1.14?

Reply:
Not really, as go.mod already reflects 1.15